Makefile

```diff
@@ -64,31 +29,9 @@ CGO_CXXFLAGS=-std=c++11
 CGO_LDFLAGS=-L$(LLVM_BUILDDIR)/lib $(CLANG_LIBS) $(LLD_LIBS) $(shell $(LLVM_BUILDDIR)/bin/llvm-config --ldflags --libs --system-libs $(LLVM_COMPONENTS))
 
 
-
 run-test: build/test
 	./build/test
```

Comment:
This rule may also be removed: it depends on `build/%` below.

Reply:
OK removed.